Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/EmojiFilter.kt

```diff
@@ -0,0 +1,32 @@
+package de.rki.coronawarnapp.ui.view
+
+import android.text.InputFilter
+import android.text.Spanned
+import com.google.android.material.textfield.TextInputEditText
+import java.lang.Character.OTHER_SYMBOL
+import java.lang.Character.SURROGATE
+
+/**
+ * Adds [InputFilter] of emojis ,returns same [TextInputEditText]
+ */
+fun TextInputEditText.addEmojiFilter(): TextInputEditText {
+    filters += EmojiFilter()
+    return this
+}
+
+private class EmojiFilter : InputFilter {
+    override fun filter(
+        source: CharSequence,
+        start: Int,
+        end: Int,
+        dest: Spanned,
+        dstart: Int,
+        dend: Int
+    ): CharSequence? {
+        for (index in start until end) {
+            val type = Character.getType(source[index]).toByte()
+            if (type in listOf(SURROGATE, OTHER_SYMBOL)) return ""
```

Comment:
Could you please add some tests to it so that we can sure that it doesn't interfere with other alphabets, e.g. russian with cyrillic letters (@AlexanderAlferov may provide you some test data), danish with [Ø](https://en.wikipedia.org/wiki/%C3%98).

Reply:
Tests https://github.com/corona-warn-app/cwa-app-android/pull/3597/commits/0f801ccae0bc949b55bffb4cdd2f5dcbce1baf49